src/rpc/rawtransaction.cpp

```diff
@@ -345,24 +362,53 @@ UniValue getrawtransaction(const UniValue& params, bool fHelp)
             "       ],\n"
-            "       \"proof\" : \"hex\",          (string) the zero-knowledge proof\n"
+            "       \"proof\" : \"hex\",            (string) the zero-knowledge proof\n"
             "       \"ciphertexts\" : [           (json array of string)\n"
             "         \"hex\"                     (string) output note ciphertext\n"
             "         ,...\n"
             "       ]\n"
             "     }\n"
             "     ,...\n"
             "  ],\n"
-            "  \"blockhash\" : \"hash\",   (string) the block hash\n"
-            "  \"confirmations\" : n,      (numeric) The confirmations\n"
-            "  \"time\" : ttt,             (numeric) The transaction time in seconds since epoch (Jan 1 1970 GMT)\n"
-            "  \"blocktime\" : ttt         (numeric) The block time in seconds since epoch (Jan 1 1970 GMT)\n"
+            "  \"valueBalance\" : x.xxx,  (numeric, optional) The sapling pool balance change in " + CURRENCY_UNIT + "\n"
+            "  \"vShieldedSpend\" : [     (array of json objects, only for version >= 3)\n"
+            "     {\n"
+            "       \"cv\" : \"hex\",           (string) Value commitment to the input note\n"
+            "       \"anchor\" : \"hex\",       (string) Merkle root of the Sapling note commitment tree\n"
+            "       \"nullifier\" : \"hex\",    (string) The nullifier of the input note\n"
+            "       \"rk\" : \"hex\",           (string) The randomized public key for spendAuthSig\n"
+            "       \"proof\" : \"hex\",        (string) A zero-knowledge proof using the spend circuit\n"
+            "       \"spendAuthSig\" : \"hex\", (string) A signature authorizing this spend\n"
+            "     }\n"
+            "     ,...\n"
+            "  ],\n"
+            "  \"vShieldedOutput\" : [            (array of json objects, only for version >= 3)\n"
+            "     {\n"
+            "       \"cv\" : \"hex\",             (string) Value commitment to the input note\n"
+            "       \"cmu\" : \"hex\",            (string) The u-coordinate of the note commitment for the output note\n"
+            "       \"ephemeralKey\" : \"hex\",   (string) A Jubjub public key\n"
+            "       \"encCiphertext\" : \"hex\",  (string) A ciphertext component for the encrypted output note\n"
+            "       \"outCiphertext\" : \"hex\",  (string) A ciphertext component for the encrypted output note\n"
+            "       \"proof\" : \"hex\",          (string) A zero-knowledge proof using the spend circuit\n"
+            "     }\n"
+            "     ,...\n"
+            "  ],\n"
+            "  \"bindingSig\" : \"hash\",          (string, optional) The sapling binding sig\n"
+            "  \"valueBalanceOrchard\" : x.xxx,  (numeric, optional) The orchard pool balance change in " + CURRENCY_UNIT + "\n"
+            "  \"valueBalanceOrchardZat\" : n,   (numeric, optional) The orchard pool balance change in zats\n"
```

Comment:
This doesn't match the format we are defining in #5894. In order to allow merging this PR first, please instead generate:
```suggestion
            "  \"orchard\" : {             (JSON object with orchard-specific information)\n"
            "     \"valueBalanceZat\" : n,     (numeric) The net value of Orchard Spends minus Outputs in " + MINOR_CURRENCY_UNIT + "\n"
            "  },\n"
```
The `orchard` object should otherwise behave like your code was already doing, i.e. present if this is a v5+ transaction, with a `valueBalanceZat` of `0` if the transaction has no Orchard component (and once #5894 is done, it would also have an empty `actions` array in this case).

cc @LarryRuane.

Reply:
I've aligned with #5894 and updated the descriptions to those suggested. I left `Spends` and `Outputs` capitalized in the descriptions. Was that intended? I also left both a `valueBalance` and a `valueBalanceZat`. OK?

I don't want to step on anyone's toes. If it's easier for me to close this in deference to #5894, I'm happy to do that.

Reply:
Thanks a lot, @buck54321 - we're planning to put out a small hotfix release tomorrow and so if #5894 is not complete in time we'll use your PR here instead.

Reply:
We decided not to proceed with the hotfix release on Friday, instead waiting until after NU5 activates today.